view.py

```diff
@@ -422,3 +558,73 @@ def create_timer_widget(canvas: tk.Canvas) -> tk.Label:
     )
     label.place(x=225, y=25, anchor="center")
     return label
+
+class GameOverView(View):
+    def __init__(
+                self, root: Root, frame: tk.Frame,
+                on_quit: Callable,
+                on_menu:Callable,
+                on_input:Callable
+        ):
+        super().__init__(root, frame)
+        size = Config.get_instance()["Game"]["Size"]
+        self.game_over_canvas = tk.Canvas(
+            self.frame,
+            width=size["Width"], height=size["Height"]
+        )
+        self.game_over_canvas.pack()
+        self.game_over_canvas.create_text(
+            225, 50,
+            text="Game\nOver",
+            font=("Arial", 50)
+        )
+
+        score = tk.StringVar()
+        score_entry = tk.Entry(self.game_over_canvas)
+
+        # Dimensions des widgets
+        self.btn_height = 100
+        self.btn_width = 200
+        
+        
+        currentdir = os.path.dirname(__file__)
+        graphics = os.path.join(currentdir, "Graphics")
+        buttons = os.path.join(graphics, "Buttons")
+
+        # Photos des widgets
+        self.quit_photo = PhotoImage(
+            file=os.path.join(buttons, "quitButton.png")
+        )
+        self.menu_photo = PhotoImage(
+            file=os.path.join(buttons, "menuButton.png")
+        )
+        
+        # Initialization des boutons
+        
+        self.btn_menu = tk.Button(
+            self.game_over_canvas,
+            image=self.menu_photo,
+            width=self.btn_width, height=self.btn_height,
+            borderwidth=0,
+            command=self.on_menu
+        )
+        self.btn_quit = tk.Button(
+            self.game_over_canvas,
+            image=self.quit_photo,
+            width=self.btn_width, height=self.btn_height,
+            borderwidth=0,
+            command=self.on_quit
+        )
+        
+        # Positionnement des widgets
+        self.btn_menu.place(
+            x=(450 - (self.btn_width*2)) / 2,
+            y=450-self.btn_height
+        )
+        self.btn_quit.place(
+            x=(450 - (self.btn_width*2)) / 2 + self.btn_width,
+            y=450-self.btn_height
+        )
+    
+    def draw(self):
+        self.pack()
```

Comment:
```suggestion
        self.pack()

```

Newline à la fin nécessaire pour les git diffs